### ISSUE TITLE
7.075MHz RTTY?

### ISSUE THREAD

**Issue by Rockscout** [s2underground/GhostNet#3]
This is inside continuous FT8 traffic on 7.074 which I doubt was intentional because my RTTY decoder grabs random characters out of an FT8 feed. Thanks for your time organising this.

**Comment by s2underground** [comment]
Yes this will be changed to a different freq (testing 7.077 now) to avoid any confusion or overlapping transmissions. There are no totally clean and unused freqs left anymore, so we just have to do the best we can.
